node/src/chain_spec.rs

```diff
@@ -155,3 +157,72 @@ fn testnet_genesis(
 		}),
 	}
 }
+
+/// Properties for Litentry.
+pub fn litentry_properties() -> Properties {
+	let mut properties = Properties::new();
+
+	properties.insert("ss58Format".into(), 31.into());
```

Comment:
What will these properties affect? I mean where can see the difference

Reply:
we can see the setting in polkadot js UI.